README.MD

```diff
@@ -121,7 +121,27 @@ async function takeScreenshot(driver, name) {
 
 start();
 ```
-## 
+
+### Cache Downloaded Binaries
+By default downloaded chromium binaries are cached in the appropriate cache directory for your operating system.
+
+You may override the cache path by setting the `NODE_CHROMIUM_CACHE` environment variable to a directory path, for example:
+
+```bash
+export NODE_CHROMIUM_CACHE=/path/to/cache/dir/
+
+# or in .npmrc like so:
+# node_chromium_cache=/path/to/cache/dir/
+```
+
+You may disable caching by setting `NODE_CHROMIUM_CACHE_DISABLE` to `true`:
```

Comment:
Last request regarding this variable, Rick: do you mind if we change it to: `NODE_CHROMIUM_DISABLE_CACHE`? For me it sounds more natural like this. What do you think?

Reply:
I can see both perspectives: 

* consistency (everything cache related starts with `NODE_CHROMIUM_CACHE` 
vs
* English verb - object word order.

I really don't mind, both are legit. Want me to change it?

Reply:
Makes sense what you say but I would like then `NODE_CHROMIUM_CACHE` variable to be renamed to `NODE_CHROMIUM_CACHE_PATH`.
Like this, it looks natural for me:
```
NODE_CHROMIUM_CACHE_PATH
NODE_CHROMIUM_CACHE_DISABLE
```
Then I'm absolutely fine with all.

Reply:
Bingo, that's a winner right there!

Reply:
Could you do this, Rick, or you want me to do it?